java/Cherepnin_Oleh/src/com/zoolatech/lecture10/tasks/Task.java

```diff
@@ -0,0 +1,30 @@
+package com.zoolatech.lecture10.tasks;
+
+import java.util.Scanner;
+
+/**
+ * Given a directory called db with 3 files:
+ * names (Ex. of content Alex, John, Maria)
+ * surnames (Scott, Fung, Chan)
+ * positions (Delivery Manager, Senior Java Dev, Product Manager)
+ * <p>
+ * When user starts app it can see in terminal list of files without content. User can type file name like
+ * "names" and see content of the file on terminal.
+ * If user type invalid filename app should write message to err stream, draw file names again and write invalid
+ * filename to a new file called "error.log" (ideally data should be appended).
+ */
+
+public class Task {
+    public static void main(String[] args) {
+        FileReader fileReader = new FileReader("src/com/zoolatech/lecture10/tasks/db/");
+        Scanner scanner = new Scanner(System.in);
+        String answer = "";
+        do {
+            System.out.println("Input file name (names, surnames, positions):");
```

Comment:
Please read all available files from directory using File object instead of hard code